package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "typescript-dom-lib-generator",
+  "name": "@microsoft/typescript-dom-lib-generator",
```

```suggestion
  "name": "@typescript/dom-lib-generator",
```

since typescript has its own scope on npm.